### ISSUE TITLE
keyboard only or mouse only

### ISSUE THREAD

**Issue by Tezonis** [debauchee/barrier#942]
Hi,

Is it possible to transfert to client keyboard control only (not mouse control) and vice versa? I can't find a way to do it.

Tezonis

**Comment by TheGP** [comment]
Yep, the problem is - if u are using a separate mouse on the client disables the keyboard as well.

**Comment by arminanton** [comment]
Is your Barrier server running on Mac?
If it is, your Mac probably have a service or app that left Secure Input active (used to protect you when entering passwords). To check if any app is using secure input, run: `ioreg -l -w 0 | grep SecureInput`. If you see anything other than blank/empty, that means there is a app/service (or many) with such feature activated.

https://www.nickjvturner.com/blog/2020/08/20/secure-input-mode
